web/components/requests/FcStatusStudyRequestBulk.vue

```diff
@@ -0,0 +1,246 @@
+<template>
```

Comment:
can we refactor this and `<FcStatusStudyRequest>` to reuse a lot of the same markup?